Fabric.Authorization.Domain/Models/Grain.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Fabric.Authorization.Domain.Models
+{
+    public class Grain : IIdentifiable, ITrackable, ISoftDelete
+    {
+        public Grain()
+        {
+            SecurableItems = new List<SecurableItem>();
+        }
+
+        public Guid Id { get; set; }
+        
+        public string Name { get; set; }
+
+        public bool IsShared { get; set; }
+
+        public ICollection<SecurableItem> SecurableItems { get; set; }
+
+        public ICollection<string> RequiredWriteScopes { get; set; }
+
+        public string Identifier => Id.ToString();
```

Comment:
i thought we were moving away from including the identifier in entities?

Reply:
I can remove that, my brain was thinking consistency :-)